tests/dummy/app/pods/details/controller.js

```diff
@@ -0,0 +1,5 @@
+import Ember from 'ember'
+
+export default Ember.Controller.extend({
+  defaultRoute: 'details.views.maps'
```

Comment:
See if Brian/Phil can throw together (5-10 minutes) some icons for these to add to the dummy app